### PR TITLE
OSDOCS-5762: clarify firewalld use language

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-5762

Link to docs preview:
Updates to text, additional resources links added
https://60520--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-firewall.html

Updates to text clarifying the likely presence of firewalld
https://60520--docspreview.netlify.app/microshift/latest/microshift_networking/microshift-firewall.html#microshift-firewall-install_microshift-firewall

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by dhellmann** [comment]
/lgtm

This looks good. I like the additional detail about what/why some traffic must be allowed.

**Comment by ShaunaDiaz** [comment]
> Would it also be helpful to include a simple package manager command to query if the firewalld package is installed

It probably would. Do you have one I can add?

**Comment by ggiguash** [comment]
> Would it also be helpful to include a simple package manager command to query if the firewalld package is installed

The command would be:
```
rpm -q firewalld
```

File: modules/microshift-firewalld-install.adoc [hunk 1]
```diff
@@ -6,10 +6,18 @@
 [id="microshift-firewall-install_{context}"]
 = Installing the firewalld service
 
-Use the following procedure to install and run the `firewalld` service for {product-title}.
+If you are using {op-system-ostree}, firewalld should already be part of your system. To use the service, you just need to configure it. The following procedure can be used if you do not have firewalld, but want to use it.
```
[the remark this answers is not in the document]
Comment (GroceryBoyJr) [reply modules/microshift-firewalld-install.adoc:9]: A small suggestion for your consideration, @ShaunaDiaz

**Review by GroceryBoyJr — COMMENTED** [review]
Only one small nit @ShaunaDiaz,  use as you see fit.  If you like it, you can rhyme it. :+1:

File: modules/microshift-firewalld-install.adoc [hunk 2]
```diff
@@ -6,10 +6,18 @@
...
 .Procedure
 
+. [Optional]. Check for firewalld on your system by running the following command:
++
+[source,terminal]
+----
+$ rpm -q firewalld
+----
+
 . To install the `firewalld` service, run the following command:
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply modules/microshift-firewalld-install.adoc:21]: I dig it. Flows better. Thx!

**Review by kelbrown20 — COMMENTED** [review]
@ShaunaDiaz Hey Shauna! Super tiny format nit before merging. Otherwise LGTM.

**Review by kelbrown20 — COMMENTED** [review]
Looks great! Thank you!

**Pull request merged**